test/parallel/test-net-connect-local-error.js

```diff
@@ -4,24 +4,27 @@ const assert = require('assert');
 const net = require('net');
 
 const server = net.createServer();
-server.listen(0);
-const port = server.address().port;
-const client = net.connect({
-  port: port + 1,
-  localPort: port,
-  localAddress: common.localhostIPv4
-});
+server.listen({port: 0}, common.mustCall(() => {
+  const port = server.address().port;
+  server.close(common.mustCall(() => {
+    const client = net.connect({
+      port: 8080,
```

Comment:
I think we should test other way around as well `{port: port, localPort: 8080}` (with a  new client)